docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
 [[connect-to-elasticsearch]]
 == Add data to {kib}
+++++
+<titleabbrev>Add data</titleabbrev>
+++++
 
-To start working with your data in {kib}, you can:
+The fastest way to add data to {kib} is to use one of the built-in options.
```

Comment:
One comment about "add data to Kibana". This statement is a little technically misleading. Kibana does not store data so you never add data to kibana. You add data to Elasticsearch or the Elastic stack. Sometimes you add data by using kibana like with sample data or file upload.

Reply:
I agree.